src/desktop/components/split_test/skipIfClientSideRoutingEnabled.ts

```diff
@@ -1,7 +1,9 @@
+import { getSplitTest } from "./splitTestContext"
+
 export const skipIfClientSideRoutingEnabled = (_req, _res, next) => {
   if (
+    getSplitTest("EXPERIMENTAL_APP_SHELL") ||
```

Comment:
Just to confirm, the DX flow is:
  - be able to toggle feature completely on or off via feature flag
  - ensure feature works totally on or off
  - staging is generally enabled (always on)
  - AB test value overrides feature flag value (and uses the new context lib)

Reply:
Yep, that's the flow!